clients/flash/air-client/src/Default.css

```diff
@@ -79,8 +79,12 @@
 /* Shared styles */
 
 global {
-	font-family : SourceSansPro;
-	color       : PropertyReference("bbbBlack");
+	font-family               : SourceSansPro;
+	color                     : PropertyReference("bbbBlack");
+	modalTransparencyBlur     : 0;
+	modalTransparency         : 0.5;
+	modalTransparencyColor    : black;
```

Comment:
Do these modalTransparency properties need to be global? Do they actually affect different classes or only one class?

Reply:
`modalTransparency` is going to be used by the `PopUpManager`, for that reason it goes globla.